index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello Microverse!</h1>
```

Comment:
- [ ] I strongly suggest adding the HTML boilerplate to your index.html file for better practices. You could read about it [here](https://www.freecodecamp.org/news/basic-html5-template-boilerplate-code-example/)
- [ ] Also, there is an issue when applying the styles because the `<h1>` it's showing with the default color. I think the issue is because you are missing the `<link rel="stylesheet" href="<YOUR_STYLESHEET_PATH_HERE>">` tag in the `<head>` section :pushpin: :wink:
![image](https://user-images.githubusercontent.com/81522435/143053147-b0a7ef11-5e1c-4e14-8bf2-b6490ee3092b.png)
![image](https://user-images.githubusercontent.com/81522435/143053184-212a3232-707c-4b1e-90e8-47bba46e627e.png)

Reply:
these comments also added to project